.github/workflows/test_and_deploy.yml

```diff
@@ -84,12 +78,12 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install -U setuptools setuptools_scm wheel twine
+          pip install -U setuptools setuptools_scm wheel twine build
       - name: Build and publish
         env:
           TWINE_USERNAME: __token__
           TWINE_PASSWORD: ${{ secrets.TWINE_API_KEY }}
         run: |
           git tag
-          python setup.py sdist bdist_wheel
+          python -m build .
```

Comment:
Can someone confirm this is fine? I can switch it back if need be (but to be honest, I'd prefer we didn't drift away from the cookiecutter plugin setup - if there are any good reasons to have something be different, we should change the cookiecutter)

Reply:
Yeah this is the new way of doing things. I thought it might need a pyproject.toml but it doesn't, works fine locally.